CMakePresets.json

```diff
@@ -11,8 +11,9 @@
       "displayName": "Default build with ninja",
       "description": "Default build using Ninja generator",
       "generator": "Ninja",
-      "binaryDir": "${sourceDir}/build-default",
+      "binaryDir": "${sourceDir}/build-${presetName}",
```

Comment:
Why is there an undocumented bugfix here?

Reply:
This just prevents duplication of binaryDir in the presets that inherit default.